xds/internal/xdsclient/xdsresource/unmarshal_rds_test.go

```diff
@@ -80,6 +108,18 @@ func (s) TestRDSGenerateRDSUpdateFromRouteConfiguration(t *testing.T) {
 				}},
 			}
 		}
+		goodUpdateWithClusterSpecifierPlugins = RouteConfigUpdate{
```

Comment:
This seems to be assuming something about the usage of `goodRouteConfigWithClusterSpecifierPlugins`.  It seems like this should be parameterized instead of hard-coded.

Reply:
It works...but now you're only testing the case where one thing ever comes out.  If you wanted to test that two things -- "A" _and_ "B" -- can be produced if they are referenced, then you'd need something else...

Reply:
Yeah, I agree, thanks for pointing it out. If I ever scale up the tests to use this, I'll change this to parameterized instead of hardcoded.